test/framework/pipelineRun_helper.go

```diff
@@ -141,7 +149,11 @@ func createPipelineRunTest(pipelineTest PipelineRunTest, run testRun) testRun {
 		run.result = fmt.Errorf("pipeline run creation failed: %q", err.Error())
 		return run
 	}
-	klog.Infof("Test: %q pipeline run created '%s/%s'", run.name, pr.GetNamespace(), pr.GetName())
+	klog.InfoS(
+		"Test - pipeline run created",
+		"testName", run.name,
+		"pipelineRun", fmt.Sprintf("%s/%s", pr.GetNamespace(), pr.GetName()),
```

Comment:
May be split namespace and name

Reply:
Pls refer [the comment](https://github.com/SAP/stewardci-core/pull/387#discussion_r1238272323) and verify the change adfbc2abfdf982d656d0b4de583541d07ca5e8a2.